src/main/java/org/jabref/gui/duplicationFinder/DuplicateResolverDialog.java

```diff
@@ -39,7 +39,7 @@ public enum DuplicateResolverResult {
         BREAK
     }
 
-    private MergeEntries mergeEntries;
+    private ThreeWayMergeView threeWayMerge;
     private final DialogService dialogService;
 
```

Comment:
Here you can use things with @Inject e.g `@Inject PreferencesService preferencesService; `t

Reply:
Injection doesn't work here either